src/scripts/items/BattleItem.ts

```diff
@@ -21,6 +21,23 @@ class BattleItem extends Item {
         return true;
     }
 
+    checkCanUse(): boolean {
```

Comment:
Should return true if it doesn't fall into any of the `if`s

Reply:
[190e4d0](https://github.com/pokeclicker/pokeclicker/pull/1791/commits/190e4d0d8790a11f51e11e6a5a1f40812cfe7a2a)